docs/configuration/configuration.md

```diff
@@ -947,6 +949,14 @@ dns_suffix: <string>
 # often the same value as dns_suffix.
 endpoint: <string>
 
+# The group(s) to filter discovery results by.
```

Comment:
This is a bit wordy, Consul has "A list of services for which targets are retrieved. If omitted, all services
 are scraped."

Reply:
Done.